models/endoscopic_tool_segmentation/docs/README.md

```diff
@@ -40,6 +40,15 @@ IoU was used for evaluating the performance of the model. This model achieves a
 #### Validation IoU
 ![A graph showing the validation mean IoU over 100 epochs.](https://developer.download.nvidia.com/assets/Clara/Images/monai_endoscopic_tool_segmentation_val_iou.png)
 
+#### TensorRT speedup
+The `endoscopic_tool_segmentation` bundle supports the TensorRT acceleration. The table below shows the speedup ratios benchmarked on an A100 80G GPU, in which the `model computation` means the speedup ratio of model's inference with a random input without preprocessing and postprocessing and the `end2end` means run the bundle end to end with the TensorRT based model. The `torch_fp32` and `torch_amp` is for the pytorch model with or without `amp` mode. The `trt_fp32` and `trt_fp16` is for the TensorRT based model converted in corresponding precision. The `speedup amp`, `speedup fp32` and `speedup fp16` is the speedup ratio of corresponding models versus the pytorch float32 model, while the `amp vs fp16` is between the pytorch amp model and the TensorRT float16 based model.
```

Comment:
Please describe the TensorRT version in the doc.

Thanks.

Reply:
I have added the torch-tensorrt version requirment to the CONTRIBUTIONG file and will add the TensorRT requirment. Should we also add these version requirement to every bundle?

Reply:
TensorRT requirement is another thing.
Here I mean maybe it's better to describe the **exact** TensorRT version you used to get this benchmark number.

Thanks.